### PR TITLE
Migrate `dev-features` to `dev` branch

### PR DESCRIPTION
**Pull request by dab246**
## Dependent

- Need merged: https://github.com/linagora/html-editor-enhanced/pull/37

## Issues Migrated

- #2644 
- #2667
- #2684
- #2717
- #2671 
- #2215

## Demo

- Mobile

[demo.webm](https://github.com/linagora/tmail-flutter/assets/80730648/43176b63-b233-4528-93d7-5a365a163259)

- Web:

https://github.com/linagora/tmail-flutter/assets/80730648/9fc18ba4-26aa-477e-af47-3f5ce414f809

### CODE REVIEW

**Comment by dab246** [comment]
### Problem on  iOS

- [ ] Part of the content is lost when replying or forwarding emails. Could be related to the problem fixed here #2749


https://github.com/linagora/tmail-flutter/assets/80730648/1b409875-f74d-4a64-b61c-f0c9023bf3a3

- [ ] The editor background appears gray when the waning dialog appears


https://github.com/linagora/tmail-flutter/assets/80730648/bd88fcc4-f8c6-43b2-a0c7-937d7b610ca8

File: pubspec.yaml [hunk 1]
```diff
@@ -55,12 +55,12 @@ dependencies:
...
     git:
       url: https://github.com/linagora/rich-text-composer.git
-      ref: master
+      ref: cnb_supported
 
   html_editor_enhanced:
     git:
       url: https://github.com/linagora/html-editor-enhanced.git
-      ref: memory_fixed_in_cnb
+      ref: migrate-dev-feature
```
Comment (hoangdat) [inline pubspec.yaml:63]: please update it

Comment (dab246) [reply pubspec.yaml]: done

**Comment by hoangdat** [comment]
@dab246 Still see the identity fetching again, even identity was fetched before

**Comment by dab246** [comment]
> @dab246 Still see the identity fetching again, even identity was fetched before
> 
> Uploading Screen Recording 2024-04-23 at 14.10.23.mov…

Fixed, please see demo video below:


https://github.com/linagora/tmail-flutter/assets/80730648/ad1d89d8-f05a-4501-af94-d0de552a6cab

**Pull request merged**